Source/WebCore/platform/text/TextFlags.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "FontFeatureValues.h"
```

Comment:
Don’t need to include the whole header just to define an argument of `RefPtr<FontFeatureValues>`. Use a forward declaration instead please.